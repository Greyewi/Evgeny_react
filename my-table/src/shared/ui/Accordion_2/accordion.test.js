import React from "react";
import rerender from 'react-test-renderer';
import { Accordion2 } from './index'

it.skip("Modal has rendered correctly", () => {
  const tree = rerender.create(<Accordion2 dataTabs={[{tabName : "name", component : <div>div</div>}]}/>).toJSON()
  expect(tree).toMatchSnapshot()
})
