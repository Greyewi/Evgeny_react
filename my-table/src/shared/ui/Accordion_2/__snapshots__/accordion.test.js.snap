// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal has rendered correctly 1`] = `
<div
  className="accordion"
>
  <div>
    <div
      className="according__title"
      onClick={[Function]}
    >
      <div>
        name
      </div>
      <img
        alt=""
        className="arrow-down"
        height={10}
        src="up-arrow.svg"
        width={10}
      />
    </div>
    <div
      className="accordion__content"
    />
  </div>
</div>
`;

exports[`Modal has rendered correctly: <div
  className="accordion"
>
  <div>
    <div
      className="according__title"
      onClick={[Function]}
    >
      <div>
        name
      </div>
      <img
        alt=""
        className="arrow-down"
        height={10}
        src="up-arrow.svg"
        width={10}
      />
    </div>
    <div
      className="accordion__content"
    />
  </div>
</div> 1`] = `
<div
  className="accordion"
>
  <div>
    <div
      className="according__title"
      onClick={[Function]}
    >
      <div>
        name
      </div>
      <img
        alt=""
        className="arrow-down"
        height={10}
        src="up-arrow.svg"
        width={10}
      />
    </div>
    <div
      className="accordion__content"
    />
  </div>
</div>
`;
